vonk/releasenotes/roadmap.rst

```diff
@@ -0,0 +1,57 @@
+.. _vonk_roadmap:
+
+Vonk Roadmap
+============
+
+This page lists the features and changes we have planned for the foreseeable future. This planning is volatile, so changes will happen. We will update this page accordingly. You are also very welcome to provide input to us on features or fixes that you would like to see prioritized. 
+
+Disclaimer: No rights can be derived from this roadmap.
```

Comment:
As in... "you had it on your roadmap" kind? Not copyrights?

Reply:
Indeed. Don't care very much about copyrights on this page :-)